src/Location/WorkingSchedule.php

```diff
@@ -319,6 +319,8 @@ public function getTimeslot(int $interval = 15, DateTime $dateTime = null, int $
     {
         $dateTime = Carbon::instance($this->parseDate($dateTime));
         $checkDateTime = $dateTime->copy();
+        if (Carbon::now()->diffInMinutes($dateTime) <= $leadTime)
+            $checkDateTime->addMinutes($leadTime);
```

Comment:
Add the logic to the `filterTimeslot` method instead, so you'll pass `$dateTime` and `$leadTime` as arguments to `filterTimeslot` then filter out timeslot when the conditional is true, so no manipulation needed on the `$dateTime` instance

Reply:
Ok, updated